src/views/virtualmachines/details/VirtualMachineNavPageTitle.tsx

```diff
@@ -25,20 +27,22 @@ import { vmTabsWithYAML } from './utils/constants';
 type VirtualMachineNavPageTitleProps = {
   instanceTypeExpandedSpec: V1VirtualMachine;
   isLoaded?: boolean;
-  name: string;
   vm: V1VirtualMachine;
 };
 
 const VirtualMachineNavPageTitle: FC<VirtualMachineNavPageTitleProps> = ({
   instanceTypeExpandedSpec,
   isLoaded,
-  name,
   vm,
 }) => {
   const { t } = useKubevirtTranslation();
   const location = useLocation();
 
-  const { vmi } = useVMI(vm?.metadata?.name, vm?.metadata?.namespace, vm?.cluster, isRunning(vm));
+  const name = getName(vm);
+  const namespace = getNamespace(vm);
+  const cluster = getCluster(vm);
+
+  const { vmi } = useVMI(getName(vm), getNamespace(vm), getCluster(vm), isRunning(vm));
```

Comment:
You created the variables above, but are using the selectors here again. Is that intentional?